canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -36,28 +37,23 @@
 		}
 	}
 	.properties-radioset-panel {
-		margin-bottom: $spacing-02;
+		margin-bottom: $spacing-01;
```

Comment:
Why was this changed?  The comment below for $spacing-01 was only for the tooltip to align it correct.  Right now things still aren't aligned.

Reply:
The tooltip is slight off.
![Screen Shot 2022-07-19 at 3 58 32 PM](https://user-images.githubusercontent.com/21126000/179847203-1bdcc8f7-3bd6-449d-8f38-c8c0b0d5e743.png)

Reply:
Put the margin back and fixed 
<img width="360" alt="image" src="https://user-images.githubusercontent.com/89609176/182492477-4814c807-a9d9-4294-9724-4feba93780c1.png">
<img width="360" alt="image" src="https://user-images.githubusercontent.com/89609176/182492511-83314487-fb9d-4b8f-a86a-02fa176fdbf2.png">
<img width="569" alt="image" src="https://user-images.githubusercontent.com/89609176/182492537-0b00b99e-20e2-4228-a3e6-6b48edb2be39.png">